### ISSUE TITLE
Incorrect normal color

### ISSUE THREAD

**Issue by JuewenPeng** [hbb1/2d-gaussian-splatting#68]
Hi, I could not visualize the correct normal map during training. Could you give me some suggestions or provide the visualization code? Thank you very much.

Here is my normal map produced by
`cv2.imwrite('normal.jpg', (0.5*rend_normal+0.5).permute(1,2,0).detach().clone().cpu().numpy() * 255)`
![image](https://github.com/hbb1/2d-gaussian-splatting/assets/38718148/3a2892cf-aa90-4606-aee3-6cd7204e825b)

**Comment by JuewenPeng** [comment]
I have tried running `render.py` without `--skip_train` to export training images, but the color of the produced normal map is still weird. 
![image](https://github.com/hbb1/2d-gaussian-splatting/assets/38718148/351696c9-ed1a-4ee8-8386-ce8108074a06)

**Comment by hbb1** [comment]
what do you mean 'weird'? I think the normal looks just fine.

**Comment by JuewenPeng** [comment]
It is inconsistent with the color of the paper.

![image](https://github.com/hbb1/2d-gaussian-splatting/assets/38718148/710afd12-f9f9-414e-af11-06d9bc54c3a1)

**Comment by hbb1** [comment]
Ahhhh, this is that we visualize the normal in view space while our default output is world space. 
To render image in view space with color like the teaser,

```python
  # get normal map
  # transform normal from view space to world space
  render_normal = allmap[2:5]
  render_normal = (render_normal.permute(1,2,0) @ (viewpoint_camera.world_view_transform[:3,:3].T)).permute(2,0,1)
  
# render the view space normal like the teaser figure
  view_normal = -allmap[2:5]
```

**Comment by Jot-De** [comment]
Hi @hbb1 ,
Thanks for great work! Could you please explain why we need to multiply allmap by -1?
